keps/sig-node/3619-supplemental-groups-policy/kep.yaml

```diff
@@ -0,0 +1,42 @@
+title: "Fine grained SupplementalGroups control"
+kep-number: 3619
+authors:
+  - "@everpeace"
+owning-sig: sig-xyz
+participating-sigs:
+  - sig-node
+status: provisional
+creation-date: 2022-10-14
+reviewers:
+  - TBD
```

Comment:
need to get these filled in!

Reply:
@mrunalp @thockin I put both of you as reviewer in kep.yaml in be119ddb0e6da67b40b16de5830a191e947cf7ce 👍